ecs-cli/modules/cli/compose/entity/service/service.go

```diff
@@ -133,6 +138,16 @@ func (s *Service) LoadContext() error {
 			s.loadBalancer.LoadBalancerName = aws.String(loadBalancerName)
 		}
 		s.role = role
+		s.loadBalancers = append(s.loadBalancers, s.loadBalancer)
+	} else if len(loadBalancerList) != 0 {
+		if len(loadBalancerList) > 5 {
```

Comment:
oh we don't do the validation for this, because this might change and differs from account to account.